openstates/nm/committees.py

```diff
@@ -0,0 +1,104 @@
+from billy.scrape.committees import CommitteeScraper, Committee
+from openstates.utils import LXMLMixin
+
+base_url = 'http://www.nmlegis.gov:8080/Committee/'
+
+
+class NMCommitteeScraper(CommitteeScraper, LXMLMixin):
+    jurisdiction = 'nm'
+
+    def scrape(self, chamber, term):
+        self.validate_term(term)
+
+        # Xpath query string format for legislative chamber committee urls
+        base_xpath = '//table[@id="MainContent_gridView{0}Committees"]' \
+                     '//a[contains(@id, "MainContent_gridView{1}Committees_link{2}Committee")]/@href'
+
+        if chamber == 'upper':
+            url = '{}Senate_Standing'.format(base_url)
+            chamber_xpath = base_xpath.format('Senate', 'Senate', 'Senate')
+
+            # Most interim committees are joint. Use 'joint' chamber as the way
+            # to collect Interim committee data
+            self.scrape('joint', term)
+
+        elif chamber == 'lower':
+            url = '{}House_Standing'.format(base_url)
+            chamber_xpath = base_xpath.format('House', 'House', 'House')
+
+        elif chamber == 'joint':
+            url = '{}Interim'.format(base_url)
+            chamber_xpath = base_xpath.format('', '', '')
+
+        self.debug('Scraping NM {0} chamber term {1} for committees listed at {2}'.format(chamber, term, url))
```

Comment:
Hey Clifton,

Generally, we try to leave debugging statements out of the source.